src/Setup/DevDivVsix/PortableFacades/PortableFacades.swr

```diff
@@ -1,29 +1,30 @@
 use vs
 
 package name=PortableFacades
-        version=1.2.0.0
+        version=1.3.0.0
```

Comment:
Binding redirects in devenv.config forward all facades to the versions we insert. I need to double check that the config file lists all of them (TODO in the comment on the PR).